chapters/methodology.tex

```diff
@@ -9,8 +9,20 @@ \subsection{Architecture}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
 \subsection{State Machine}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
-\subsection{User Interface}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+\begin{figure}
+  \includegraphics[width=\linewidth]{figures/state_diagramm.jpg}
+  \caption{The state diagram.}
+  \label{fig:state_diagramm}
+\end{figure}
+
 
+Figure \ref{fig:state_diagramm} shows the states and transitions of the state machine.
+A state change can be invoked throw three different kind of actions: An user action (for example pressing a button), a timeout after a specified couple of seconds or an significant analyser result (face detection or smile detection).

```

Comment:
throw -> through
Maybe action -> interaction?
Maybe couple of seconds -> timeframe?